tests/suites/test_suite_psa_crypto.function

```diff
@@ -40,11 +40,14 @@
 #define ASSERT_OPERATION_IS_INACTIVE(operation) TEST_ASSERT(operation.id == 0)
 
 #if defined(PSA_WANT_ALG_JPAKE)
-int ecjpake_operation_setup(psa_pake_operation_t *operation,
-                            psa_pake_cipher_suite_t *cipher_suite,
-                            psa_pake_role_t role,
-                            mbedtls_svc_key_id_t key,
-                            size_t key_available)
+/**
+ * TODO: This function is currently unused. What shall we do with it?
```

Comment:
Please remove it. We have git for the history.

Reply:
Done in a9d40d7